### PR TITLE
[Enhancement] tablet sink support add chunk large than 2GB

### PR DESCRIPTION
**Pull request by luohaha**
## Why I'm doing:
In data ingestion, we transfer chunk data between BEs via the rpc `tablet_writer_add_chunk` and `tablet_writer_add_chunks`. And when chunk size large than 2GB, serialization of`PTabletWriterAddChunksRequest` and`PTabletWriterAddChunkRequest` will fail because of protobuf limitation. Error message like this:

```
google/protobuf/message_lite.cc:410 starrocks.PTabletWriterAddChunkRequest exceeded maximum protobuf size of 2GB: 2324260102
```

## What I'm doing:
When chunk size large than 2GB (can be modified by the session variable `rpc_http_min_size`), we will use brpc attachment to bypass protobuf serialization.

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [x] Yes, this PR will result in a change in behavior.
- [ ] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [x] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: gensrc/proto/doris_internal_service.proto [hunk 1]
```diff
@@ -64,6 +64,8 @@ service PBackendService {
     rpc transmit_chunk_via_http(starrocks.PHttpRequest) returns (starrocks.PTransmitChunkResult);
     rpc tablet_writer_add_chunk(starrocks.PTabletWriterAddChunkRequest) returns (starrocks.PTabletWriterAddBatchResult);
     rpc tablet_writer_add_chunks(starrocks.PTabletWriterAddChunksRequest) returns (starrocks.PTabletWriterAddBatchResult);
+    rpc tablet_writer_add_chunk_via_http(starrocks.PHttpRequest) returns (starrocks.PTabletWriterAddBatchResult);
```
Comment (trueeyu) [inline gensrc/proto/doris_internal_service.proto:67]: Already deprecated， no need add here

**Review by trueeyu — DISMISSED** [review]

**Review by rickif — APPROVED** [review]

File: be/src/common/utils.h [hunk 2]
```diff
@@ -49,4 +53,35 @@ void set_request_auth(T* req, const AuthInfo& auth) {
     }
 }
 
+template <typename T>
+static void serialize_to_iobuf(butil::IOBuf& iobuf, T* proto_obj) {
```
[the remark this answers is not in the document]
Comment (luohaha) [reply be/src/common/utils.h]: Done

**Review by trueeyu — APPROVED** [review]

**Review by meegoo — APPROVED** [review]

**Comment by luohaha** [comment]
https://github.com/Mergifyio backport branch-3.3

**Comment by luohaha** [comment]
https://github.com/Mergifyio backport branch-3.3

**Comment by luohaha** [comment]
ignore backport check: 3.1.13

**Comment by luohaha** [comment]
ignore backport check 3.1.14

**Pull request merged**